baseplate/server/thrift.py

```diff
@@ -25,12 +23,7 @@ def __init__(self, processor, *args, **kwargs):
                 THeaderClientType.UNFRAMED_BINARY,
             ]
         )
-        super(GeventServer, self).__init__(*args, **kwargs)
-
-    def serve_forever(self, stop_timeout=None):
-        signal.signal(signal.SIGINT, lambda sig, frame: self.stop())
-        signal.signal(signal.SIGTERM, lambda sig, frame: self.stop())
-        super(GeventServer, self).serve_forever(stop_timeout=stop_timeout)
```

Comment:
What happened here?

Reply:
The base server stuff stopped using `serve_forever` in a4144dfb8b772b3d477da614eeb4145e1ee81273